nexus-sdk/src/main/java/io/nexusrpc/OperationInfo.java

```diff
@@ -15,57 +15,62 @@ public static Builder newBuilder(OperationInfo info) {
     return new Builder(info);
   }
 
-  private final String id;
+  private final String token;
   private final OperationState state;
 
-  private OperationInfo(String id, OperationState state) {
-    this.id = id;
+  private OperationInfo(String token, OperationState state) {
+    this.token = token;
     this.state = state;
   }
 
-  /** ID of the operation. */
+  /** Get the operation token. */
   public String getId() {
```

Comment:
What is the posture here about leaving things around but deprecated as opposed to removal? For instance, this getter remains but the setter on the builder does not. (and I'd guess the deprecated pieces will be removed soon anyways)

Reply:
I didn't plan on leaving any but the header key. I only plan on leaving the header key for older servers

Reply:
I just missed these